apps/style/common.scss

```diff
@@ -644,6 +644,10 @@ input[type="radio"] {
   text-align: justify;
   -ms-text-justify: distribute-all-lines;
   text-justify: distribute-all-lines;
+
+  .legacy-share-view & {
+
+  }
```

Comment:
Nothing. That was a mistake, leftover from an earlier prototype. I realized I could hide that area by not rendering the reference area. Removed